cypress/integration/prospects/profile.js

```diff
@@ -16,5 +16,10 @@ describe('Prospects profile info', () => {
       // go back to the prospect list page and continue
       cy.go('back')
     }
+    
+    const index = Math.floor(Math.random() * 10) + 10
```

Comment:
this should be included in a different test (you should call the it function again)